packages/forms/frontend/sirius-components-forms/src/propertysections/ButtonPropertySection.tsx

```diff
@@ -58,6 +66,15 @@ const useStyle = makeStyles<ButtonStyleProps>()(
       flexDirection: 'row',
       gap: theme.spacing(2),
     },
+    loadingIndicator: {
+      color: theme.palette.divider,
+      marginLeft: theme.spacing(2),
```

Comment:
Use a proper layout in the parent HTML element instead of making this HTML element aware of its surrounding elements.

Reply:
I'm not sure why there should even be a margin left here given that there is a gap already